packages/opentelemetry-basic-tracer/src/BasicTracer.ts

```diff
@@ -22,7 +22,7 @@ import {
   NoopSpan,
   randomSpanId,
   randomTraceId,
-  INVALID_SPAN_CONTEXT,
+  NOOP_SPAN_INSTANCE,
```

Comment:
Why the change? I personally preferred `NOOP_SPAN` as I think "instance" is redundant.

Reply:
Hmm, I will update the same PR without "instance" suffix.

Reply:
Done.